course_access_groups/permissions.py

```diff
@@ -45,9 +46,20 @@ def is_organization_staff(user, course):
     course_org_ids = OrganizationCourse.objects.filter(
         course_id=str(course.id),
         active=True,
-    ).values('organization_id')
+    ).values_list('organization_id', flat=True)
```

Comment:
Please refactor into the following more explicit and manageable code:

I recommend the following implmenetaiton which simplifies a lot of the checks that aren't needed.

```py
# course access groups
def is_organization_staff(user, course):
    if not user.is_active:
        return False

    try:
        course_organization = tahoe_sites.api.get_organization_by_course(course_id=course.id)
    except MultipleObjectsReturned, Organization.DoesNotExist:
        # Safely handle the exception errors by assuming the user is not a staff.
        return False
    return is_active_admin_on_organization(user=user, organization=course_organization)
```

The helper above would need another API helper on `tahoe_sites`:

```py
# tahoe sites
def get_organization_by_course(course_id):
    return Organization.filter(
        pk__in=OrganizationCourse.objects.filter(
            course_id=str(course.id),
            active=True,
        )
    ).get()
```

Reply:
Done